examples/plot_derivative.py

```diff
@@ -0,0 +1,79 @@
+"""
+Derivatives
+===========
+This example demonstrates how to use a set of derivative operators, namely
+:py:class:`pylops_mpi.basicoperators.MPIFirstDerivative` and
+:py:class:`pylops_mpi.basicoperators.MPISecondDerivative`.
+
+The derivative operators are really useful when dealing with
```

Comment:
`The derivative operators are really useful when dealing with inverted models.` -> `Derivative operators are commonly used when solving inverse problems within regularization terms aimed at enforcing smooth solutions`

Reply:
Done